### ISSUE TITLE
Error Code 400 with NodeJs and 500 with Python

### ISSUE THREAD

**Issue by pauldj54** [MicrosoftDocs/bot-docs#1666]
**Describe the bug**
I unsuccessfully followed both, the nodejs and python tutorials.
When I run the app using nodejs and open the url (http://localhost:3978/api/messages) in my browser I get: `{"code":"MethodNotAllowed","message":"GET is not allowed"}`
When I try to connect the bot emulator:
![image](https://user-images.githubusercontent.com/706754/79390660-696c6e80-7f70-11ea-9a3e-0e569d96bfbf.png)

I tried also with python, similar result:
When I open localhost:3978 on the browser I got a "500 internal server error" and when connect to the bot emulator the same result as above.

**To Reproduce**
1.Just follow the tutorial

**Expected behavior**
According to the tutorial it should work.

**Additional context**
I tested in two Conda environments with python 3.6 and 3.7 respectively
Node Js version v12.6.2
OS: Windows 10 Enterprise

**Comment by pauldj54** [comment]
Since I am working in my company laptop, even when I am connected to my home network and I disabled the proxy settings (env variables, npm, git, conda, etc) I suspected there could be something wrong with networks, firewall rules, etc.
Therefore I started a fresh VM in Azure and tried again.
This time everything ran smoothly.
However, I would like to know which settings I have to modify in order to make it work in my local machine. 
Best, 
Paul
